README.md

```diff
@@ -1,98 +1,120 @@
   ```bash
   export PATH=$PATH:/usr/local/sbin
   sudo rabbitmq-server
   ```
 
-* To see RabbitMQ dashboard visit: [`http://localhost:15672/`][rabbit-MQ2]
-* The user name will be `guest` and password `guest`.
+### RabbitMQ dashboard
+* The RabbitMQ dashboard is available at [`http://localhost:15672/`][rabbit-MQ2]
+* username: `guest` / password: `guest`
+* AIPScan connects to the RabbitMQ queue on port `:5672`.
+
 
-### Celery
+## Celery
+Celery was automatically installed into the AIPscan project as a Python module dependency during the initial AIPscan requirements import command:  `pip install -r requirements/base.txt`
 
-* In another terminal window, from the AIPscan root directory, start a Celery
-worker: `celery worker -A AIPscan.worker.celery --loglevel=info`
+To start up Celery workers that are ready to receive tasks from RabbitMQ:
+* Open a new terminal tab or window.
+* Navigate to the AIPscan root project directory.
+* Activate the Python virtualenv in the AIPscan project directory so that the Celery dependency gets automatically loaded: 
+  `source venv/bin/activate`
+* Enter the following command:  
+  `celery worker -A AIPscan.worker.celery --loglevel=info`
+* You should see terminal output similar to this to indicate that the Celery task queu is ready:
 
-## Usage
+![screencap6](screencaps/aipscan_celery_hello_world.png)
 
-### Connecting to a storage service and initiating AIPScan's use
+# Usage
 
-* Go to [`localhost:5000`][usage-1] in a browser.
+* Ensure that the Flask Server, RabbitMQ server, and Celery worker queu are up and running.  
+* Go to [`localhost:5000`][usage-1] in your browser.
 * Select "New Storage Service"
 * Add an Archivematica Storage Service record, including API Key, eg.
 `https://amdemo.artefactual.com:8000`
 * Select "New Fetch Job"
-* When the Fetch Job completes, select "View AIPs" button, "AIPs" menu, or
-"Reports" menu.
-
-## Screenshots
-
-### Example AIPScan fetch job
-
-![screencap1](screencaps/aipscan_fetch_job.png)
-
-### Viewing an AIP in AIPScan
-
-![screencap2](screencaps/aipscan_view_aip.png)
-
-### Selecting a report to run in AIPScan
-
-![screencap3](screencaps/aipscan_select_report.png)
-
-### Demonstration of a scatter chart report
-
-![screencap4](screencaps/aipscan_scatterplot_report.png)
+* Check the black and green terminal to confirm that AIPscan successfully connected to the Archivematica Storage Service, that it received the lists of available packages from Archivematica, and that it has begun downloading and parsing the AIP METS files.
+* Sit back and have a coffee, tea, beer, wine or whatever rocks your boat. This could take a while (i.e. a few hours) depending on the total number of AIPs in your Storage Service and the size of your METS xml files. Therefore, if you have the option, it is recommended that you test AIPscan on a smaller subset of your full AIP holdings first. This should help you estimate the total time to run AIPscan against all packages in your Storage Service.
```

Comment:
I think this is more casual than we would typically put in our READMEs. It did get a good laugh out of me though :smile:

Reply:
Hmm, yeah, you can tell I was getting a little punch at this stage of the updates :-)